### PR TITLE
fix vl creation when copy vl of kind busbreaker

### PR DESCRIPTION
**Pull request by Walid-Sahnoun**

### CODE REVIEW

**Review by ayolab — APPROVED** [review]

File: src/components/graph/menus/modification-list-item.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const ModificationListItem = ({
     const classes = useStyles();
     const studyUuid = useSelector((state) => state.studyUuid);
     const currentNode = useSelector((state) => state.currentTreeNode);
-    const [computedValues, setComputedValues] = useState({});
+    const [computedValues, setComputedValues] = useState();
```
Comment (FranckLecuyer) [inline src/components/graph/menus/modification-list-item.js:70]: why this change ? is there no risk of regression ?

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/components/graph/menus/modification-list-item.js (hunk 1 above)
Comment (Walid-Sahnoun) [reply src/components/graph/menus/modification-list-item.js:70]: this change because there is an error of translation showed in console
like this, i fixed and there is no regression,
i tested multiple modifications and there is no problem

**Review by FranckLecuyer — APPROVED** [review]
